### ISSUE TITLE
Secrets for containerEnv

### ISSUE THREAD

**Issue by g-kartik** [devcontainers/cli#772]
Currently the --secrets-file sets the secret environment variables as `remoteEnv`. A support for `containerEnv` shall ensure that all the running processes inside the devcontainer can access them.

**Comment by samruddhikhandale** [comment]
I wonder if it's very similar to the ask in https://github.com/orgs/devcontainers/discussions/119. 

From https://github.com/orgs/devcontainers/discussions/119#discussioncomment-8761462

> Not sure if a static env file would cover your case, but there is a feature request for supporting .env files: https://github.com/microsoft/vscode-remote-release/issues/3007.

**Comment by g-kartik** [comment]
Yes it is similar to https://github.com/microsoft/vscode-remote-release/issues/3007
